src/renderer/atlas/AtlasEngine.h

```diff
@@ -347,6 +349,7 @@ namespace Microsoft::Console::Render
 
             SmallObjectOptimizer& operator=(SmallObjectOptimizer&& other) noexcept
             {
+                this->~SmallObjectOptimizer();

```

Comment:
_yikes_

Reply:
wow is this a valid implementation of an assignment operator?

Reply:
_double yikes_

Reply:
I believe so. If it makes yourself more comfortable I can use `std::destruct_at` and `std::construct_at` (which is valid to do), but they basically do the exact same thing. 😅

The current code leaks memory however so this fix is somewhat important.